.env

```diff
@@ -0,0 +1 @@
+THOR_MERGE="code -d $1 $2"
```

Comment:
Is this something you had used locally to help with the rails app:update merging? Not sure this is something we want in the repo.

Reply:
I don't think a .env file should be in git at all (it was also intended for the bash environment, to do the `rails app:update` as you said). I'll look into how to remove it.

Reply:
I pushed a commit to remove this file.